sparkplug_plugin/sparkplug_b_input.go

```
@@ -0,0 +1,1135 @@
+
+	// Add birth-death sequence if available from node state
+	s.stateMu.RLock()
+	if state, exists := s.nodeStates[deviceKey]; exists {
+		msg.MetaSet("spb_bdseq", fmt.Sprintf("%d", state.bdSeq))
+	}
+	s.stateMu.RUnlock()
+
+	return msg
+}
+
+func (s *sparkplugInput) createMessageFromPayload(payload *sproto.Payload, msgType, deviceKey string, topicInfo *TopicInfo, originalTopic string) *service.Message {
```

golangci-lint (1.64.8)

func `(*sparkplugInput).createMessageFromPayload` is unused

(unused)